app/web_ui/src/routes/(app)/settings/manage_tools/[project_id]/tool_servers/[tool_server_id]/+page.svelte

```diff
@@ -322,3 +337,10 @@
     {/if}
   </AppPage>
 </div>
+
+<DeleteDialog
+  name="Tool Server"
+  bind:this={delete_dialog}
+  {delete_url}
```

Comment:
you make delete_url dynamic with `$:` but just attach it statically so the component won't get dyanmic updates.

You probably want:
```
bind:delete_url={`/api/projects/${project_id}/tool_servers/${tool_server_id}`}
```